File: Lab01/Python_Lang/Demo05.py
#
#  Created by NDHuy on 05/04/2020
#  Copyright 2020 by SEstudio
#  Input a integer number. After that print character of number
#

def number_to_string(num):
    switcher = {
        0 : "Zero",
        1 : "One", 
        2 : "Two",
        3 : "Three",
        4 : "Four",
        5 : "Five", 
        6 : "Six", 
        7 : "Seven", 
        8 : "Eight",
        9 : "Nine"
    }
    return switcher.get(num, "Invalid character of number")


def main():
    num = int(input("Input a integer number: "))
    print(number_to_string(num))


if __name__ == "__main__":
    main()

File: Lab01/C_Lang/src/Demo05.cpp
//
//  Created by NDHuy on 05/04/2020
//  Copyright 2020 by SEstudio
//  Input a integer number. After that print character of number
//

#include <stdio.h>

int main()
{
    int num; 
    printf("Input a integer number: ");
    scanf("%d", &num);

    switch(num)
    {
        case 0:
            printf("Zero\n");
            break;
        case 1:
            printf("One\n");
            break;
        case 2:
            printf("Two\n");
            break;
        case 3:
            printf("Three\n");
            break;
        case 4: 
            printf("Four\n");
            break;
        case 5:
            printf("Five\n");
            break;
        case 6:
            printf("Six\n");
            break;
        case 7:
            printf("Seven\n");
            break;
        case 8:
            printf("Eight\n");
            break;
        case 9:
            printf("Nine\n");
            break;
        default:
            printf("Invalid character of number\n");
    }

    return 0;
}